### PR TITLE
[Vue] remove angularjs from the Morpheus plugin

### PR DESCRIPTION
**Pull request by diosmosis**
### Description:

**This PR is based off of #19391.**

Changes:
* Rewrite demo.twig to use and display Vue snippets.
* Remove angularjs directive use from twig templates.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/Morpheus/templates/contentBlock.twig [hunk 1]
```diff
@@ -2,8 +2,8 @@
     <div class="card-content">
         {% if title is defined and title %}
             <h2 class="card-title"
-                  {% if rate is defined and rate and rate is true %}piwik-enriched-headline="{{ title|e('html_attr') }}"
-                  {% elseif rate is defined and rate and rate %}piwik-enriched-headline="{{ rate|e('html_attr') }}"{% endif %}
+                  {% if rate is defined and rate and rate is true %}vue-entry="CoreHome.EnrichedHeadline"
+                  {% elseif rate is defined and rate and rate %}vue-entry="CoreHome.EnrichedHeadline"{% endif %}
```
Comment (sgiehl) [inline plugins/Morpheus/templates/contentBlock.twig]: Does this change make sense? Before it was either assigned `title` or `rate` to the headline. While I'm not sure if the old angular component was handling that at all, I don't think the new vue component does, or am I wrong?
Also the second condition has a duplicate `and rate`, which is useless.

File: plugins/Morpheus/vue/src/Demo/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  extends: [
+    '../../../../../.eslintrc.js',
+  ],
+  rules: {
+    'max-len': 'off',
+  },
+}
```
Comment (sgiehl) [inline plugins/Morpheus/vue/src/Demo/.eslintrc.js]: Is this file still needed? Doesn't look like there are any lines that are too long.

**Review by sgiehl — CHANGES_REQUESTED** [review]
Merged in the latest changes. Left two comments. Once they are clarified, this should be good to merge. Might be possible some screenshots needs another update...

File: plugins/Morpheus/templates/contentBlock.twig (hunk 1 above)
Comment (diosmosis) [reply plugins/Morpheus/templates/contentBlock.twig]: The old directive doesn't handle a piwikEnrichedHeadline attribute value so I don't think this code actually did anything before. So my reasoning is it should be fine to remove it.

File: plugins/Morpheus/vue/src/Demo/.eslintrc.js (hunk 2 above)
Comment (diosmosis) [reply plugins/Morpheus/vue/src/Demo/.eslintrc.js]: It's gone now so I'm guessing it's not.

**Comment by diosmosis** [comment]
@sgiehl applied review feedback

**Review by sgiehl — APPROVED** [review]
